src/event_loop.rs

```diff
@@ -178,11 +178,11 @@ impl<T> EventLoopWindowTarget<T> {
     }
 
     /// Returns the primary monitor of the system.
+    ///
+    /// Returns `None` if it can't identify any monitor as a primary one.
```

Comment:
Shouldn't the documentation here match the one on Window?

Reply:
Well, for Wayland bits - yes, but for `ios` there's no need to, since it's always called on the main thread due to event loop limitations, since it's always on a mean thread already.